### PR TITLE
[v13] Document DynamoDB backend billing_mode option

### PR DESCRIPTION
**Pull request by lxea**
Backport #28672 to branch/v13

### CODE REVIEW

File: docs/pages/reference/backends.mdx [hunk 1]
```diff
@@ -581,6 +591,13 @@ To enable these options you will need to update the IAM Policy for Teleport.
 }
 ```
 
+<Admonition
+  type="warning"
+  title="pay_per_request with autoscaling"
+>
+  If `pay_per_request` is set to true, the settings for read/write capacity and auto scaling will be ignored.
```
Comment (zmb3) [inline docs/pages/reference/backends.mdx:598]: It's not a Boolean, so it can't be set to true. Please fix this.

**Review by zmb3 — CHANGES_REQUESTED** [review]

File: docs/pages/reference/backends.mdx (hunk 1 above)
Comment (r0mant) [reply docs/pages/reference/backends.mdx:598]: Looks like the [original PR](https://github.com/gravitational/teleport/pull/28672) has the same issue so @lxea please send a follow up to fix it in master as well.

Comment (zmb3) [reply docs/pages/reference/backends.mdx:598]: Yeah, sorry I missed the original. I approved with a few suggestions before these changes were made and a re-review was never requested.

File: docs/pages/reference/backends.mdx [hunk 2]
```diff
@@ -544,6 +549,11 @@ teleport:
     # auto_scaling is used to optionally enable (and define settings for) auto scaling.
     # default: false
     auto_scaling:  [true|false]
+    # Enables either On-Demand or Provisioned billing for the DynamoDB table. Set when Teleport creates the table.
```
[the remark this answers is not in the document]
Comment (r0mant) [reply docs/pages/reference/backends.mdx:552]: Doesn't look like this was updated here?
```suggestion
    # Enables either Pay-Per-Request or Provisioned billing for the DynamoDB table. Set when Teleport creates the table.
```

**Review by r0mant — APPROVED** [review]
with a comment

File: docs/pages/reference/backends.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (lxea) [reply docs/pages/reference/backends.mdx]: ah, there were two that needed fixing and I missed this one :facepalm:

**Review by zmb3 — APPROVED** [review]

**Pull request merged**